pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -35,7 +35,9 @@ let
   options = {
 
     debug = {
-      DEBUG_INFO                = if (features.debug or false) then yes else no;
+      # Necessary for BTF
+      DEBUG_INFO                = yes;
```

Comment:
```suggestion
      DEBUG_INFO                = if (features.debug or (whenAtLeast "5.2" yes) or false) then yes else no;
```

Reply:
Did it slightly differently because I couldn't get it to work with `if`s.